cmake_modules/win32.cmake

```diff
@@ -18,18 +18,20 @@ if (WIN32)
         # set(QT_PATH "C:/msys64/clang64")
         find_program(
             TOOL_WINDEPLOYQT
-                windeployqt windeployqt6
+                windeployqt windeployqt-qt6 windeployqt6
                 REQUIRED NO_CACHE
                 HINTS "${QT_PATH}/bin"
         )
-        message("windeployqt found: ${TOOL_WINDEPLOYQT}")
+        message("-- windeployqt found: ${TOOL_WINDEPLOYQT}")
 
-        string(APPEND CMAKE_FIND_LIBRARY_SUFFIXES ";.dll")
+        set(CMAKE_FIND_LIBRARY_SUFFIXES ";.dll" PARENT_SCOPE)
+        #string(APPEND CMAKE_FIND_LIBRARY_SUFFIXES ";.dll")
```

Comment:
Is this proper? Something tells me this isn't. But if it works for you and cartographic I won't pick at it.

Reply:
Not sure if it is proper. Im basically deleting everything and setting .dll as the only suffix for windows. Because MSYS2 is unix based this was an easy way to get rid of the dll.a extension